dxp/build_process.md

```diff
@@ -60,7 +60,7 @@ The CLI can also run [webpack](https://webpack.js.org/) as part of the process i
 
 #### 6. Liferay npm bundler
 
-Running [liferay-npm-bundler](https://github.com/liferay/liferay-js-toolkit/tree/master/packages/liferay-npm-bundler) allows your project's dependencies and source code to be converted to [AMD}(https://en.wikipedia.org/wiki/Asynchronous_module_definition) so that they can be deployed as an OSGi package later. So, Liferay Portal will be able to load the files of your module and its dependencies when requested.
+Running [liferay-npm-bundler](https://github.com/liferay/liferay-js-toolkit/tree/master/packages/liferay-npm-bundler) allows your project's dependencies and source code to be converted to [AMD](https://en.wikipedia.org/wiki/Asynchronous_module_definition) so that they can be deployed as an OSGi package later. So, Liferay Portal will be able to load the files of your module and its dependencies when requested.
```

Comment:
That `}` ... 😱

![noooooo](https://user-images.githubusercontent.com/7074/93433315-eeb0d200-f8c6-11ea-8eea-229701b575b9.gif)

Reply:
![there there](https://media.giphy.com/media/VuWtVHkMjrz2w/giphy.gif)

Reply:
😂😂😂